.github/workflows/labeler.yml

```diff
@@ -7,6 +7,6 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@v3
+    - uses: actions/labeler@v4
```

Comment:
`v4` is the newest and yet GitHub did not upgrade the target NodeJS version of their code... :shrug: